### ISSUE TITLE
PROD: We have people creating accounts on behalf of other people. We need to prevent this.

### ISSUE THREAD

**Issue by pauljanes** [krico/jas#734]
J is creating user accounts so that she can add guests to her classes without making them sign up for an account. This is wrong as it means the user is never agreeing to our AGB.

We need to use email verification to prevent prevent people creating accounts on behalf of others (ie using their email).

It isnt critical at the moment because those who create an account via Google or Facebook are not even asked to sign our AGB. But when we close the Google and FB account creation AGB loophole then we need to close this one too.

Basically every user in our system needs to have agreed to our AGB.

**Comment by wszarmach** [comment]
Duplicate of #640
